src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -276,13 +304,26 @@ export class RelayFeeCalculator {
   async relayerFeeDetails(
     amountToRelay: BigNumberish,
     tokenSymbol: string,
-    tokenPrice?: number,
-    _originRoute?: ChainIdAsString,
-    _destinationRoute?: ChainIdAsString
+    originRoute: ChainIdAsString,
+    destinationRoute: ChainIdAsString,
```

Comment:
We don't need these as underlined variables anymore. This is a holdover from when this function was stubbed.